hackclub.com.yaml

```
@@ -2021,7 +2021,11 @@ onboard:
 oneditor:
   - ttl: 600
     type: CNAME
-    value: oneditor.netlify.app.
+    value: oneditor.netlify.app.   
```

Sorry to bother you, just noticed that you added some extraneous whitespace here. Could it be removed? Otherwise, you should be good to go!
```suggestion
    value: oneditor.netlify.app.
```

---

spurious change should be reverted